src/api/application.md

```diff
@@ -438,11 +438,11 @@ Adjusts template whitespace handling behavior.
 
 - **Details**
 
-  Vue removes / condenses whitespaces in templates to produce more efficient compiled output. The default strategy is "condense", with the following behavior:
+  Vue removes / condenses whitespace in templates to produce more efficient compiled output. The default strategy is "condense", with the following behavior:
 
-  1. Leading / ending whitespaces inside an element are condensed into a single space.
-  2. Whitespaces between elements that contain newlines are removed.
-  3. Consecutive whitespaces in text nodes are condensed into a single space.
+  1. Leading / ending whitespace inside an element is condensed into a single space.
```

Comment:
I believe this case is correct as we're speaking about plural form here

Reply:
Whitespace is usually an uncountable noun https://english.stackexchange.com/a/25371 thus the singular usage.

Reply:
Swapped it for `whitespace characters` so we can keep it plural.